src/Core/Services/Implementations/OrganizationService.cs

```diff
@@ -1403,6 +1403,41 @@ public async Task DeleteUserAsync(Guid organizationId, Guid userId)
             }
         }
 
+        public async Task DeleteUsersAsync(Guid organizationId, IEnumerable<Guid> organizationUsersId,
+            Guid? deletingUserId)
+        {
+            var orgUsers = await _organizationUserRepository.GetManyAsync(organizationUsersId);
+            var filteredUsers = orgUsers.Where(u => u.OrganizationId == organizationId);
+
+            if (!filteredUsers.Any())
+            {
+                throw new BadRequestException("Users invalid.");
+            }
+
+            var confirmedOwners = (await GetConfirmedOwnersAsync(organizationId)).ToList();
+            var confirmedOwnersIds = confirmedOwners.Select(u => u.Id);
+            if (!confirmedOwnersIds.Except(organizationUsersId).Any())
+            {
+                throw new BadRequestException("Organization must have at least one confirmed owner.");
+            }
+
+            foreach (var orgUser in filteredUsers)
+            {
+                // TODO: We should replace this call with `DeleteManyAsync`.
```

Comment:
Should we not go ahead and implement a DeleteMany repo method as part of this PR?

Reply:
It is implemented in the https://github.com/bitwarden/server/pull/1311.